src/client/Microsoft.Identity.Client/MsalErrorMessage.cs

```diff
@@ -362,6 +362,8 @@ public static string TokenTypeMismatch(string requestTokenType, string responseT
                 requestTokenType, responseTokenType);
         }
 
+        public const string TokenTypeMissing = "The response from the token endpoint does not contain token_type parameter. This happens if the Identity Provider (AAD, B2C, ADFS, etc.) did not include the token type in the response. In case of ADFS, verify the configuration or consider upgrading to the latest version. ";
```

Comment:
the latest version of ADFS? (not the library?)

Reply:
I made this message similar to the `TokenTypeMismatch` above. Yes, in this case it's probably a ADFS configuration issue or a ADFS bug.

Reply:
nit: also might consider making it "AccessTokenType" to match the other one as well.

Reply:
Which `the other one` are you referring to?

Reply:
sorry, i'll work on being more clear -> like this `_requestParams.AuthenticationScheme.AccessTokenType` and actually call it `AccessTokenType` because that's what it is. so this would be `AccessTokenTypeMissing` - just a thought.

Reply:
Thanks for clarifying. Updated.